docs/DafnyRef/Topics.md

```diff
@@ -600,24 +604,30 @@ _inductive datatype_.
 Least solutions like $g^{\downarrow}$ have been given the
 name _least predicate_.
 
-Let's prove $g^{\downarrow}(x) \;\Longrightarrow\; 0 \leq x \;\wedge\; x \textrm{ even}$.
+Let's prove $g^{\downarrow}(x) \Longrightarrow 0 \leq x \wedge x \text{ even}$.
 We split our task into two cases, corresponding to which of the two
 proof rules in [the inductive rules](#g-ind-rule) was the
 last one applied to establish $g^{\downarrow}(x)$.  If it was the left-hand rule, then $x=0$,
 which makes it easy to establish the conclusion of our proof goal.  If it was the
 right-hand rule, then we unfold the proof tree one level and obtain $g^{\downarrow}(x-2)$.
 Since the proof tree for $g^{\downarrow}(x-2)$ is smaller than where we started, we invoke
-the _induction hypothesis_ and obtain $0 \leq (x-2) \;\wedge\; (x-2) \textrm{ even}$, from which
+the _induction hypothesis_ and obtain $0 \leq (x-2) \wedge (x-2) \textrm{ even}$, from which
 it is easy to establish the conclusion of our proof goal.
 
 Here's how we do the proof formally using [the least exists definition](#eq-least-is-exists).  We massage the
 general form of our proof goal:
 
-|   | $f^{\uparrow}(x) \;\Longrightarrow\; R$                                                    |
-| = | &nbsp;&nbsp;&nbsp;&nbsp; { [the least exists definition](#eq-least-is-exists) }                        |
-|   | $(\exists k \bullet\; { {}^{\flat}\kern-1mm f}_k(x)) \;\Longrightarrow\; R$                              |
-| = | &nbsp;&nbsp;&nbsp;&nbsp; { distribute $\;\Longrightarrow\;$ over $\exists$ to the left } |
-|   | $\forall k \bullet\; ({ {}^{\flat}\kern-1mm f}_k(x) \;\Longrightarrow\; R)$                              |
+<p style="text-align: center;">
+$$
+\begin{array}{lll}
+    & f^{\uparrow}(x) \;\Longrightarrow\; R  & \\
+  = &  & \textrm{ (the least exists definition) }    \\
+    & (\exists k \bullet\; { {}^{\flat}\kern-1mm f}_k(x)) \;\Longrightarrow\; R    &     \\
+  = &  & \text{distribute} \;\Longrightarrow\; \text{over} \;\exists\; \text{to the left}  \\
+    & \forall k \bullet\; ({ {}^{\flat}\kern-1mm f}_k(x) \;\Longrightarrow\; R)        &       
+\end{array}
+$$
+</p>
 
 The last line can be proved by induction over $k$.  So, in our case, we prove
 ${ {}^{\flat}\kern-1mm g}\_k(x) \Longrightarrow 0 \leq x \wedge x \textrm{ even}$ for every $k$.
```

Comment:
There's a math expression later in this paragraph that isn't being rendered correctly (`$x=0$`). (I can't add a comment there directly since it appears outside the diff.)

Reply:
Check the preview like the others.